packages/hardhat-ethers/src/dist/src/signer-with-address.ts

```diff
@@ -1 +1 @@
-export { SignerWithAddress } from "../../signers";
+export { CustomEthersSigner as SignerWithAddress } from "../../signers";
```

Comment:
Two questions here:

1. I don't like the `CustomEthersProvider` and `CustomEthersSigner` name. What do you think about `HardhatEthersProvider` and `HardhatEthersSigner`?
2. Should we keep exporting `SignerWithAddress` as an alias for backwards compatibility or should we drop it?